tests/network/general/test_bridge_marker.py

```diff
@@ -169,6 +173,5 @@ def test_bridge_marker_devices_exist_on_different_nodes(
         multi_bridge_attached_vmi.wait_until_running(timeout=_VM_NOT_RUNNING_TIMEOUT, logs=False)
 
     # validate the exact reason for VMI startup failure is missing bridge
-    pod = multi_bridge_attached_vmi.virt_launcher_pod
     for bridge in bridge_networks:
```

Comment:
Why do you need to iterate over bridge_networks? pass both to `_assert_failure_reason_is_bridge_missing` and check there if one of them is in the error (if you still want to check the error message).
otherwise, iiuc, it may be that the 2nd bridge is in the error so the function will be called twice.

Reply:
please do this in a follow up pr